### DISCUSSION TITLE
Allow more users into the beta

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#3630] [Codespaces] [answered]
To get more feedback on it, and it sounds like a fantastic idea I think people should be let into the beta sooner rather then like months later, because I have been waiting a while.

Anyway, can't wait to try it.

**Comment by PranamBhat** [comment]
**True!!!**

**Comment by 2percentsilk** [comment, marked as answer]
Hi @Milo123459,

Thanks for your feedback. We know there's a lot of excitement for the beta (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We'll add more folks as soon as we can!
